src/extension/background-script/index.ts

```diff
@@ -9,14 +10,24 @@ import state from "./state";
 
 let isFirstInstalled = false;
 
+declare global {
```

Comment:
```suggestion
// when debugging is enabled in development mode a window.debugAlby object is defined that can be used within the console. This is the type interface for that
declare global {
```

do we need a comment there?